### PR TITLE
Fix null attribute_changes crash in Activities component

### PR DESCRIPTION
**Pull request by patrickweh**
## Summary

- Adds null-safe operator to `attribute_changes` access in `Activities::loadData()` - the column is nullable but `->toArray()` was called without null check
- Adds type hints to closure parameters in the same method

Reported from production (sbm-verlag.de) where an activity log entry had `attribute_changes = NULL`.

## Summary by Sourcery

Handle activities with nullable attribute_changes and strengthen type safety in the Activities component while adding regression coverage.

Bug Fixes:
- Prevent a crash in the Activities Livewire component when an activity log entry has a null attribute_changes column.

Enhancements:
- Add explicit Activity model and closure parameter type hints in the Activities component for better type safety.

Tests:
- Add a Livewire test ensuring Activities handles activity_log entries with null attribute_changes without errors.

### CODE REVIEW

File: src/Livewire/Support/Activities.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
 use Livewire\Attributes\Renderless;
 use Livewire\Component;
 use Livewire\WithPagination;
+use Spatie\Activitylog\Models\Activity;
```
Comment (SirSplasch) [inline src/Livewire/Support/Activities.php:15]: Are you intentionally not using the flux Activity model?

**Review by SirSplasch — CHANGES_REQUESTED** [review]

File: src/Livewire/Support/Activities.php (hunk 1 above)
Comment (patrickweh) [reply src/Livewire/Support/Activities.php:15]: yes

**Review by SirSplasch — APPROVED** [review]
